src/components/Person.js

```diff
@@ -5,150 +5,136 @@ import store from 'store2';
 import { updated_niceties_spinlock } from "./People";
 
 
-const Person = React.createClass({
-    getInitialState: function() {
-        let textValue = '';
-        let anonValue = false;
-        let noReadValue = false;
-        let dataPerson;
-        let foundPerson = false;
-        for (var i = 0; i < this.props.fromMe.length; i++) {
-            if (this.props.fromMe[i].target_id === this.props.data.id) {
-                dataPerson = this.props.fromMe[i];
-                foundPerson = true;
-                break;
-            }
-        }
-        let dateUpdated;
-        if (foundPerson) {
-            if (dataPerson.date_updated === null) {
-                dateUpdated = '';
-            } else {
-                dateUpdated = dataPerson.date_updated;
-            }
+class Person extends React.Component {
+
+  constructor(props) {
+    super(props)
```

Comment:
The references to `this.props` in the constructor can be converted into references to the `props` argument. The call to `super(props)` should set `this.props`, though; just noting this as a step the [`class` codemod takes](https://github.com/reactjs/react-codemod#explanation-of-the-new-es2015-class-transform-with-property-initializers).